### PR TITLE
Changing DNS resolver from google to OpenNIC

### PR DESCRIPTION
**Pull request by albpal**
This PR is related to the work done in https://gitcoin.co/issue/dappnode/DAppNode/31/1421

### CODE REVIEW

File: build/bind/eth.hosts [hunk 1]
```diff
@@ -7,19 +7,19 @@ eth			IN SOA	eth. admin.eth. (
...
+
 $ORIGIN dnp.dappnode.eth.
-*			    A	172.33.1.3
 my.admin		A	172.33.1.9
 my.bind			A	172.33.1.2
 my.ethchain		A	172.33.1.6
-my.ethforward	A	172.33.1.3
-my.installer	A	172.33.1.7
+my.ethforward		A	172.33.1.3
+my.installer		A	172.33.1.7
```
Comment (vdo) [inline build/bind/eth.hosts]: Do you mind changing `installer` to `dappmanager` here?

**Review by vdo — CHANGES_REQUESTED** [review]

File: build/bind/named.conf [hunk 2]
```diff
@@ -5,10 +5,10 @@ options {
 		127.0.0.1/32;
 		172.33.0.0/16;
 	};
-
+	
 	forwarders {
-		8.8.8.8;
-		8.8.4.4;
+		185.121.177.177;
+		169.239.202.202;
```
Comment (vdo) [inline build/bind/named.conf]: I think it would be nice to have a third resolver here just in case OpenNIC ones don't work

Comment (albpal) [reply build/bind/named.conf]: The problem with this option is that if you add a resolver that doesn't resolve OpenNIC domains, you cannot guarantee that you will always resolve OpenNIC domains. Bind doesn't resolve in list order as /etc/resolv.conf does.

**Comment by vdo** [comment]
Thanks @albpal . Your PR looks much better now. It will be merged once we test it.

**Comment by Shelpin** [comment]
Hey @vdo !has this been tested implemented?? please let me know when it is done to include it in user doc...thanks!!!

**Comment by dapplion** [comment]
Will review this in the next meeting

**Comment by Shelpin** [comment]
Implemented DNScrypt...Thanks @albpal !!

**Pull request closed without merging**